cdk/lib/stack.ts

```diff
@@ -563,8 +565,8 @@ export class PinBoardStack extends GuStack {
         [ENVIRONMENT_VARIABLE_KEYS.databaseHostname]: databaseHostname,
       },
       monitoringConfiguration: {
-        noMonitoring: true,
-        // toleratedErrorPercentage: 0 TODO consider alarming on errors (need to provide sns topic which is sad since GuAlarm finds it for you)
+        snsTopicName: ALARM_SNS_TOPIC_NAME,
+        toleratedErrorPercentage: 0,
```

Comment:
would you want `okAction`?

Reply:
ooo yes I do, thanks.
(LOL, I added the `okAction` to GuCDK 🤦  - https://github.com/guardian/cdk/pull/1752)